client/lib/plans/constants.js

```diff
@@ -705,7 +803,89 @@ export const FEATURES_LIST = {
 		getDescription: () => i18n.translate(
 			'Custom polls, surveys, ratings, and quizzes for the ultimate in customer and reader engagement.'
 		)
-	}
+	},
+	[ FEATURE_CORE_JETPACK ]: {
+		getSlug: () => FEATURE_CORE_JETPACK,
+		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getDescription: () => i18n.translate(
+			'Stats, customization, and promotion tools'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_BASIC_SECURITY_JETPACK ]: {
+		getSlug: () => FEATURE_BASIC_SECURITY_JETPACK,
+		getTitle: () => i18n.translate( 'Basic Security' ),
+		getDescription: () => i18n.translate(
+			'Brute force protection, monitoring, secure logins, updates.'
```

Comment:
Alternate string suggestion: 
* `translate( 'Jetpack offers free <a %s>brute force protection</a>, <a %s>uptime monitoring</a>, and <a %s>secured logins</a>. We've already blocked 3.8 billion malicious login attempts.' )`&mdash; translations: **18**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-plans-tab)